Meraki.Api/Data/DhcpServerPolicyAlerts.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Meraki.Api.Data;
+
+/// <summary>
+/// additional information/config options for things like alerts/arp inspection
+/// </summary>
+[DataContract]
+public class DhcpServerPolicyAlerts
+{
+	/// <summary>
+	/// Whether Email alerts are enabled for the DhcpServerPolicy
+	/// </summary>
+	[ApiAccess(ApiAccess.ReadUpdate)]
+	[DataMember]
+	public DhcpServerPolicyAlertEnabledState Email { get; set; } = new();
```

Comment:
Also make "Email" nullable in case it's not set or relevant, so we don't have it on one side and not the other of a compare when it's not relevant.